### DISCUSSION TITLE
Running out of memory

### DISCUSSION THREAD

**Discussion by GratonR** [Nevysha/Cozy-Nest#165] [General]
Since I've installed cozy nest my browser crashes. I get a message saying "Out of memory" and I need to reload the page.
The problem is that by doing that I lose all of my prompts !
I tried on Brave and Opera GX, both same results. Brave seems to crash later tho.

**Comment by Nevysha** [comment]
I'll improve memory usage in next version. Not sure if it will solve your problem entirely but still.
